src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/ViewModels/Tournaments/TournamentViewModel.cs

```diff
@@ -205,10 +205,15 @@ public bool IsDivisionsCountMax
         /// </summary>
         /// <param name="tournament"> Domain object </param>
         /// <returns> View model object </returns>
+        public string AlarmMessage { get { return !this.IsArchived ? "Tournament was successfully archived": "Tournament was successfully unarchived"; } }
```

Comment:
What is the purpose here?

Reply:
that was written before us 
when we archve tournament than we saw alarm message that we archive tournament

but now , admin is able to activate tournament and  i thought that it would be better to write the same alarm message when admin activate our tournament

Reply:
There was no such property